nservicebus/upgrades/5to6.md

```diff
@@ -610,4 +610,18 @@ The BSON serializer build into the core has been removed. Use the [Newtonsoft se
 
 [MSMQ Subscription Authorization](/nservicebus/msmq/subscription-authorisation.md) is now done by the `SubscriptionAuthorizer` delegate at configuration time and not the `IAuthorizeSubscriptions` interface.
 
-snippet: 5to6-MsmqSubscriptionAuthorizer
\ No newline at end of file
+snippet: 5to6-MsmqSubscriptionAuthorizer
+
+## Synchronous request-response
+
+Synchronous request-response capability, also known as the callbacks, has been removed from the `NServiceBus` package in V6. There is a separate `NServiceBus.Callbacks` NuGet package that needs to be installed in the requester endpoint in order to use the feature. The responding endpoint does not need any additional packages.
+
+snippet: 5to6-Callbacks
+
+In V6 we also changed the way callback messages are routed. In V6 callback routing is based on unique endpoint instance ID. In order to continue using callbacks, you need to assign this IDs on the requester side
+
+snippet: 5to6-Callbacks-InstanceId
+
+NOTE: This ID needs to be stable and it should never be hardcoded. An example approach might be reading it from the configuration file or from the environment (e.g. role ID in Azure).
```

Comment:
I'd say the cost is higher than potential gain in this small note, but I'll keep that in mind (didn't know about that feature).